Segment/Classes/SEGWebhookIntegration.m

```diff
@@ -0,0 +1,208 @@
+#import <Foundation/Foundation.h>
+#import "SEGWebhookIntegration.h"
+#import "SEGHTTPClient.h"
+#import "SEGState.h"
+#import "SEGAnalyticsUtils.h"
+#import "SEGUtils.h"
+
+
+@interface SEGWebhookIntegration : NSObject <SEGIntegration>
+@property(nonatomic, strong) SEGHTTPClient *client;
+@property(nonatomic, strong) NSString *webhookUrl;
+@property(nonatomic, strong) NSString *name;
+@property(nonatomic, strong) SEGAnalytics *analytics;
+@property (nonatomic, strong) dispatch_queue_t serialQueue;
+
+- (instancetype)initWithAnalytics:(SEGAnalytics *)analytics httpClient:(SEGHTTPClient *)client webhookUrl:(NSString *)webhookUrl name:(NSString *)name;
+@end
+
+@implementation SEGWebhookIntegration
+
+- (instancetype)initWithAnalytics:(SEGAnalytics *)analytics httpClient:(SEGHTTPClient *)client webhookUrl:(NSString *)webhookUrl name:(NSString *)name {
+    if (self = [super init]) {
+        _name = name;
+        _analytics = analytics;
+        _client = client;
+        _webhookUrl = webhookUrl;
+        _serialQueue = seg_dispatch_queue_create_specific("io.segment.analytics.webhook", DISPATCH_QUEUE_SERIAL);
```

Comment:
There is a warning (we may not have it enabled) but implicit self is usually a warning. If you change these to:
`self->_name = name; // etc` that should prevent the warning from ever popping up.

Reply:
ah ok! will change 👍

Reply:
is there a difference between
```self->_name```
and 
```self.name``` ?